.github/workflows/search.yaml

```
@@ -3,12 +3,18 @@ name: Algolia Search
   schedule:
     - cron: '0 10 * * *'
+  # TODO: Remove trigger from push event when search index scraper is fixed
+  push:
+    branches:
+      - '7-fix-search'
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)